gradle/libs.versions.toml

```diff
@@ -5,6 +5,7 @@ multiplatform-settings = "1.2.0"
 nexus-publish = "2.0.0-rc-1"
 android-minSdk = "21"
 android-compileSdk = "34"
+ios-minSdk = "15"
```

Comment:
Can we target 13?

Reply:
I decreased it to 14; if we can confirm that it works with 13 at any time, we can change it again